uk/template_extending/README.md

```diff
@@ -131,4 +131,4 @@
 
 Ось і все! Перевірте, чи ваш сайт все ще коректно працює :)
 
-> Якщо з'явилася помилка `TemplateDoesNotExists`, то це говорить про те, що файлу `blog/base.html` не існує і ви маєте в консолі запущений `runserver`, спробуйте зупинити його (натиснувши Ctrl+C - кнопки Control і C buttons разом) і перезавантажити командою `python manage.py runserver`.
```

Comment:
Yes, as I could (without actually knowing the language) conclude that the English word "button" was used in the meaning of "key" there, too. Though someone who actually knows Ukrainian should probably review that. Although unlikely, maybe in Ukrainian the foreign word "button" is used to mean what would be "key" in English.